policyengine_us/parameters/gov/hhs/medicaid/eligibility/categories/infant/age_range/in_ca.yaml

```
@@ -0,0 +1,18 @@
+description: Age range defining an infant for Medicaid in California.
+brackets:
+  - threshold:
+      2015-01-01: 0
+    amount:
+      2015-01-01: true
+  - threshold:
+      2015-01-01: 2
+    amount:
+      2015-01-01: false
+metadata:
+  type: single_amount
+  rate_unit: bool
```

```suggestion
  amount_unit: bool
```

Rate unit is used in `marginal_rate` parameters